content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md

```diff
@@ -65,10 +65,10 @@ You can enable access to the Dashboard using the `kubectl` command-line tool,
 by running the following command:
 
 ```
-kubectl proxy
+kubectl proxy --port=8001
 ```
 
-Kubectl will make Dashboard available at [http://localhost:8001/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard:/proxy/](http://localhost:8001/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard:/proxy/).
+Kubectl will make Dashboard available at [http://localhost:8001/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard-kong-proxy:443/proxy/](http://localhost:8001/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard-kong-proxy:443/proxy/).
```

Comment:
Where does the extra name come from? I don't see a mention of Kong elsewhere in this tutorial.

Reply:
https://github.com/kubernetes/dashboard/blob/0efb014bcaad8c838eefc41b8fe54cf3d44c8e23/docs/user/accessing-dashboard/README.md?plain=1#L30

Reply:
I don't think this URI change is valid.

Reply:
> I don't think this URI change is valid.

I am a k8s newbie. When I deploy the dashboard based on the documentation, I find that `http://localhost:8001/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard:/proxy/` does not redirect to the dashboard web page and `kubectl -n kubernetes-dashboard get svc` doesn't find the service named kubernetes-dashboard. I found the official dashboard documentation `https://github.com/kubernetes/dashboard/blob/0efb014bcaad8c838eefc41b8fe54cf3d44c8e23/docs/user/accessing-dashboard/README.md?plain=1#L30` works. So I guess the documentation in this repository is outdated.

To confirm, I tried deploying dashboard in a ready-to-use k8s test environment `https://killercoda.com/playgrounds/course/kubernetes-playgrounds/one-node-4GB` and tested it with curl.
![CleanShot_2024-10-08_at_14 36 01@2x](https://github.com/user-attachments/assets/407182a0-8115-4b94-ab19-f99ae540a52e)
![CleanShot_2024-10-08_at_14 38 10@2x](https://github.com/user-attachments/assets/f7b5939e-9d4d-4147-941b-8994f5eb5d0e)

If there is something wrong with my operating procedure, please point it out. Or advise why you think this URI change is illegal.

Reply:
The problem is that there are many ways to deploy a kubernetes cluster. There is no indication (as far as I know) that we need Kong to run the kubernetes-dashboard.
By following different deployment guides, you get different cluster settings.
The URL change indicates that Kong is deployed, but Kong is not a required component.
Please help consult the dashboard project for confirmation.